resources/prosody-plugins/mod_end_conference.lua

```diff
@@ -0,0 +1,6 @@
+local endconference_component = module:get_option_string('end_conference_component', 'endconference.'..module.host);
```

Comment:
I think it's no longer necessary to split the component modules like this, see the breakout rooms module ;-)

Reply:
Migrated code into a single file.